scripts/skills/perks/perk_rf_discovered_talent.nut

```diff
@@ -1,6 +1,6 @@
 this.perk_rf_discovered_talent <- ::inherit("scripts/skills/skill", {
 	m = {
-		IsApplied = false
+		AttributesRolled = []
```

Comment:
This could also be just a U8 byte where each bit position from 0 - 7 represents, whether the corresponding attribute from `::Const.Attributes` was already picked.

That will 
- simplify the serialization code
- probably improve performance slightly

But might complicate 
- inclusion of additional attributes in the future
- readability of the non-serialized code

Have you considered this option?

Reply:
That is an intriguing suggestion indeed. I think the performance concern is negligible here. But I think the drawback of not being able to work out of the box with additional attributes added by other mods is a big one. So I would prefer an approach that is not limited by that.